src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.dialog.html

```diff
@@ -1,4 +1,4 @@
-﻿<ng-form name="dtgeForm" novalidate>
+﻿<ng-form id="dtgeForm" name="dtgeForm" novalidate>
```

Comment:
ah just spotted that myself, i added it earlier whilst trying to make it work, and found a better way. You're right, its not needed now